docs/contributor/pocs/otlp-logs.md

```diff
@@ -104,3 +105,63 @@ In the used configuration, we move the original log record to the **original** a
+
+The figure below shows the different plugins that are configured in the processing pipeline. Important to mention is the use of the batch processor in the gateway, which introduces asynchronicity to the pipeline and causes that backpressure is not propagated back to the agent. To minimize the risk of log loss due to the batch processors properties, a persistent exporter queue was introduced in the gateway, which uses a persistent volume to buffer logs in case of a backend failure.
+
+![Otel Collector Setup](./assets/otlp-logs.drawio.svg)
+
+To deploy the OpenTelemetry Collector agent and gateway, perform the following steps:
+
+1. Create a SAP Cloud Logging instance as described above
+
+1. Create a persistent volume claim (PVC):
+
+   ```bash
+   kubectl apply -n otel-logging -f ./assets/otel-gateway-pvc.yaml
+   ```
+
+1. Deploy the gateway:
+
+   ```bash
+   helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+   helm install -n otel-logging logging open-telemetry/opentelemetry-collector -f ./assets/otel-log-gateway-values.yaml
+   ```
+
+1. Deploy the agent:
+
+   ```bash
+   helm install -n otel-logging logging open-telemetry/opentelemetry-collector -f ./assets/otel-log-agent-values.yaml
+   ```
+
+### Evaluation
+
+To evaluate the buffering and backpressure capabilities of the described OpenTelemetry Collector setup, we tested the following scenarios and observed the described behavior:
+
+* **Outage of the OTLP backend**
+
+  Log records cannot be shipped from the gateway to the OTLP backend (SAP Cloud Logging). Once the configured queue limit is reached, log records will be dropped from the queue. The enqueue errors are not propagated back to other pipeline elements due to the asynchronicity introduced by the batch processor.
+
+* **Broken connectivity between the agent and the gateway**
+
+  Log records cannot be exported by the agent to the gateway using the OTLP protocol. The exporter queue on the agent will up to its maximum size and then start rejecting new records. This enqueue error is propagated to the filelog receiver, which eventually stops reading new logs. Log loss is avoided until the log retention of the kubelet removes old logs.
+
+### Conclusions
+
+The evaluation of the two failure scenarios showed that the OpenTelemetry Collector can similar guarantees about the prevention of log loss as the current Fluent Bit setup. When using a batch processer, using a persistent output queue and with that increasing the queue capacity, helps to prevent data loss. Splitting that processing pipeline to agent and gateway allows to use a PVC for the exporter queue and with that give it a large capacity without the risk that the node file-system fills up.
```

Comment:
```suggestion
The evaluation of the two failure scenarios showed that the OpenTelemetry Collector can have similar guarantees about the prevention of log loss as the current Fluent Bit setup. When using a batch processor, using a persistent output queue and with that increasing the queue capacity, helps to prevent data loss. Splitting that processing pipeline to agent and gateway allows to use a PVC for the exporter queue and with that give it a large capacity without the risk that the node file-system fills up.
```

May be re-writing this sentence would help:  `When using a batch processor, using a persistent output queue and with that increasing the queue capacity, helps to prevent data loss.` its difficult to follow